### PR TITLE
Japanese - Tips for Designing

### PR DESCRIPTION
**Pull request by jaykishigami**

### CODE REVIEW

File: _tips/designing.ja.md [hunk 1]
```diff
@@ -0,0 +1,669 @@
...
+last_updated: 2024-03-29   # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+resource:
+  ref: /tips/  # Do not change this
+
+navigation:
+  previous: /tips/writing/ # Do not change this
+  next: /tips/developing/ # Do not change this
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
```
Comment (remibetin) [inline _tips/designing.ja.md:18]: @jaykishigami Please remove the hash sign and the space from the beginning of this line.

File: _tips/designing.ja.md [hunk 2]
```diff
@@ -0,0 +1,669 @@
...
+footer: >
+  <p><strong>Date:</strong> Minor update 9 January 2019. Updated 15 April 2016. First published September 2015.</p>
+  <p><strong>Editors:</strong> <a href="https://www.w3.org/People/kevin">Kevin White</a>, <a href="https://www.w3.org/People/shadi">Shadi Abou-Zahra</a>, and <a href="https://www.w3.org/People/Shawn">Shawn Lawton Henry</a>謝辞</p>
+<p>    <a href="https://www.w3.org/WAI/EO/">教育と援助のためのWG (EOWG)</a>によって開発され.
+    <a href="https://www.w3.org/WAI/DEV/">WAI-DEV プロジェクト</a>からのサポートを受けるとともに, ヨーロッパ委員会の <abbr title="Information Society Technologies">IST</abbr> プログラムの援助を受けた｡</p>
+
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
```
Comment (remibetin) [inline _tips/designing.ja.md:51]: @jaykishigami Please translate "Summary"

File: _tips/designing.ja.md [hunk 3]
```diff
@@ -0,0 +1,669 @@
...
+
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+
+
+{::options toc_levels="2" /}
+
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Page Contents" class="full" %}
```
Comment (remibetin) [inline _tips/designing.ja.md:64]: @jaykishigami Please translate "Page Contents"

File: _tips/designing.ja.md [hunk 4]
```diff
@@ -0,0 +1,669 @@
...
+
+前景テキストは背景色と十分なコントラストが必要です｡これには画像上のテキスト､背景グラデーション､ボタン､その他の要素が含まれます｡ロゴや､写真に偶然写り込んだテキストなどの付随的なテキストには適用されません｡以下のリンクは､WCAGが要求する最小コントラスト比の詳細と､コントラストをチェックする方法を提供しています｡「コントラスト比」は､より技術的に正しい用語「輝度コントラスト比」の短縮版です｡.
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="<strong>例:</strong> コントラスト比" class="example" %}
+{:/}
+
+<div class="two-column">
+  <figure>
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Wrong" %}不十分な場合t</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:84]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 5]
```diff
@@ -0,0 +1,669 @@
...
+    <div>
+      <p style="color: #999;">文字と背景のコントラストが十分でないと文字が読めない人もいる｡また､明るい色（高輝度）は読めず､低輝度が必要な人もいる｡.</p>
+    </div>
+  </figure>
+  <figure>
+    <figcaption>{% include_cached icon.html name="check-circle" label="OK" %} 十分な場合</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:90]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 6]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [コントラスト（最小） 1.4.3](/WAI/WCAG21/quickref/#contrast-minimum) ([Understanding 1.4.3](/WAI/WCAG21/Understanding/contrast-minimum))
```
Comment (remibetin) [inline _tips/designing.ja.md:106]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 7]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+## 色だけで情報を伝えない
+
+色は情報を伝達するのに役立つが､色だけで情報を伝達すべきではない｡要素を区別するために色を使用する場合は､色の知覚に依存しない追加的な識別も行いましょう｡例えば､フォームの必須項目を示すために色に加えてアスタリスクを使用したり､グラフの領域を区別するためにラベルを使用したりしましょう
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="Example: Using color to convey meaning" class="example" %}
```
Comment (remibetin) [inline _tips/designing.ja.md:123]: @jaykishigami The `title` value needs translation.

File: _tips/designing.ja.md [hunk 8]
```diff
@@ -0,0 +1,669 @@
...
+{:/}
+
+<div class="color-alone two-column">
+  <figure class="reference fail">
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Wrong" %}色だけを使った場合</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:128]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 9]
```diff
@@ -0,0 +1,669 @@
...
+          <label for="name1">名前</label> <input type="text" id="name1" name="name">
+        </div>
+        <div class="row">
+          <label class="required" style="color: #992929;" for="email1">メール</label> <input type="text" id="email1" name="email">
+        </div>
+      </form>
+    </div>
+  </figure>
+  <figure class="reference pass">
+    <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}色と記号を使った場合</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:142]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 10]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: 色だけで何かを参照する場合" class="example" %}
+{:/}
+<div class="color-alone two-column">
+  <figure class="question fail">
+    <figcaption>{% include_cached icon.html name="ex-circle" label="Wrong" %}色だけの場合</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:166]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 11]
```diff
@@ -0,0 +1,669 @@
...
+          <input id="answer-blue1" name="answer1" value="blue" type="radio"> <label for="answer-blue1">青</label><br>
+          <input id="answer-red1" name="answer1" value="red" type="radio"> <label for="answer-red1">赤</label><br>
+          <input id="answer-yellow1" name="answer1" value="yellow" type="radio"> <label for="answer-yellow1">黄色</label><br>
+          <input id="answer-pass1" name="answer1" value="pass" type="radio"> <label for="answer-pass1">わからない</label>
+        </fieldset>
+      </form>
+    </div>
+  </figure>
+  <figure class="question pass">
+    <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}色と数字</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:190]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 12]
```diff
@@ -0,0 +1,669 @@
...
+    <div>
+      <svg version="1.1" width="153" height="154">
+        <g>
+          <path d="m2.06902,49.92402l91.82715,-48.2264l-42.35869,75.85164l-49.46846,-27.62524z" fill="#09610D" />
+          <text fill="#f4f4f4" y="51" x="44"><tspan>1</tspan><tspan dx="100" style="opacity:0">Green triangle: Angles are 30°, 60°, 90°</tspan></text>
```
Comment (remibetin) [inline _tips/designing.ja.md:195]: @jaykishigami Please translate the text in this line: `Green triangle: Angles are 30°, 60°, 90°` and the following lines.

File: _tips/designing.ja.md [hunk 13]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [色の使用 1.4.1](/WAI/WCAG21/quickref/#use-of-color) ([Understanding 1.4.1](/WAI/WCAG21/Understanding/use-of-color))
```
Comment (remibetin) [inline _tips/designing.ja.md:227]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 14]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: 異なるリンク状態に対する独自のスタイル" class="example" %}
+{:/}
+
+<div class="focus two-column">
+  <div>
+    <figure>
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}テキストからリンクを目立たせる</figcaption>
+      <div>
+        <p>マウスが使えず､ウェブページを<a href="javascript:return false;" aria-label="Link is styled to stand out from main text">キーボードだけで操作</a>する人もいる｡</p>
```
Comment (remibetin) [inline _tips/designing.ja.md:249]: @jaykishigami Please translate the `aria-label` value.

File: _tips/designing.ja.md [hunk 15]
```diff
@@ -0,0 +1,669 @@
...
+        <p>ユーザーがキーボードを使ってすべてのインタラクティブな要素に到達でき､どの要素にフォーカスされているかが明確に分かることが重要です｡</p>
+        <p>目に見えるキーボード・フォーカスは､ウェブ・ページをタブで移動するときに移動するボーダーやハイライトで示される｡</p>
+      </div>
+    </figure>
+  </div>
+  <div>
+    <figure class="hover">
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %} マウスホーバースタイル</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:257]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 16]
```diff
@@ -0,0 +1,669 @@
...
+リンクやボタンなどのインタラクティブな要素には､識別しやすいように明確なスタイルを用意しましょう｡たとえば､マウスホバー､キーボードフォーカス､タッチスクリーン起動時にリンクの外観を変更します｡インタラクティブ要素のスタイルとネーミングは､ウェブサイト全体で一貫して使用するようにしましょう｡
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: 異なるリンク状態に対する独自のスタイル" class="example" %}
+{:/}
+
+<div class="focus two-column">
+  <div>
+    <figure>
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}テキストからリンクを目立たせる</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:247]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 17]
```diff
@@ -0,0 +1,669 @@
...
+        <p>ユーザーがキーボードを使ってすべてのインタラクティブな要素に到達でき､どの要素にフォーカスされているかが明確に分かることが重要です｡</p>
+        <p>目に見えるキーボード・フォーカスは､ウェブ・ページをタブで移動するときに移動するボーダーやハイライトで示される｡</p>
+      </div>
+    </figure>
+  </div>
+  <div>
+    <figure class="hover">
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %} マウスホーバースタイル</figcaption>
+      <div>
+        <p style="text-align: center; position:relative;"><a href="javascript:return false;" aria-label="Link is styled with a subtle background to highlight hover state" style="color: #850BAC; background-color: rgba(253, 247, 15, 0.3)">keyboard to navigate</a><img src="../img/pointer.png" style="position: absolute; left: 50%; top:1em;" width="20" alt=""></p>
```
Comment (remibetin) [inline _tips/designing.ja.md:259]: @jaykishigami Please translate the `aria-label` value.

File: _tips/designing.ja.md [hunk 18]
```diff
@@ -0,0 +1,669 @@
...
+      </div>
+    </figure>
+    <figure class="keyboard">
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %}キーボードフォーカスのスタイル</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:263]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 19]
```diff
@@ -0,0 +1,669 @@
...
+      <div>
+        <p style="text-align: center; position:relative;"><a href="javascript:return false;" aria-label="Link is styled with a stronger version of the hover style to make it stand out" style="color: #850BAC; background-color: #fdf70f">keyboard to navigate</a></p>
```
Comment (remibetin) [inline _tips/designing.ja.md:265]: @jaykishigami Please translate the `aria-label` value.

File: _tips/designing.ja.md [hunk 20]
```diff
@@ -0,0 +1,669 @@
...
+      </div>
+    </figure>
+    <figure class="active">
+      <figcaption>{% include_cached icon.html name="check-circle" label="OK" %} タッチまたはクリックスタイル</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:269]: @jaykishigami Please translate the `label` value.

File: _tips/designing.ja.md [hunk 21]
```diff
@@ -0,0 +1,669 @@
...
+      <div>
+        <p style="text-align: center; position:relative;"><a href="javascript:return false;" aria-label="Link is given a strong styling to show an action will occur" style="background: #850BAC; color: #f4f4f4">keyboard to navigate</a><img src="../img/pointer.png" style="position: absolute; left: 50%; top:1em;" width="20" alt=""></p>
```
Comment (remibetin) [inline _tips/designing.ja.md:271]: @jaykishigami Please translate the `aria-label` value.

File: _tips/designing.ja.md [hunk 22]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [目に見えるフォーカス方法 2.4.7](/WAI/WCAG21/quickref/#focus-visible) ([Understanding 2.4.7](/WAI/WCAG21/Understanding/focus-visible))
```
Comment (remibetin) [inline _tips/designing.ja.md:286]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 23]
```diff
@@ -0,0 +1,669 @@
...
+  * [一貫した識別 3.2.4](/WAI/WCAG21/quickref/#consistent-identification) ([Understanding 3.2.4](/WAI/WCAG21/Understanding/consistent-identification))
```
Comment (remibetin) [inline _tips/designing.ja.md:287]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 24]
```diff
@@ -0,0 +1,669 @@
...
+
+## 明確で一貫性のあるナビゲーション・オプションの提供
+ウェブサイト内のページ間のナビゲーションは､一貫性のあるネーミング､スタイル､配置にすること｡サイト内検索やサイトマップなど､複数のナビゲーション方法を用意すること｡トピックパスや明確な見出しなど､自分のいる位置を示す手がかりを提供することで､ユーザーがウェブサイトやページのどこにいるのかを理解を助けます｡
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [一貫性のあるナビゲーション 3.2.3](/WAI/WCAG21/quickref/#consistent-navigation) ([Understanding 3.2.3](/WAI/WCAG21/Understanding/consistent-identification))
```
Comment (remibetin) [inline _tips/designing.ja.md:304]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 25]
```diff
@@ -0,0 +1,669 @@
...
+  * [様々な方法 2.4.5](/WAI/WCAG21/quickref/#multiple-ways) ([Understanding 2.4.5](/WAI/WCAG21/Understanding/multiple-ways))
```
Comment (remibetin) [inline _tips/designing.ja.md:305]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 26]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [ラベルまたは説明書 3.3.2](/WAI/WCAG21/quickref/#labels-or-instructions) ([Understanding 3.3.2](/WAI/WCAG21/Understanding/labels-or-instructions))
```
Comment (remibetin) [inline _tips/designing.ja.md:358]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 27]
```diff
@@ -0,0 +1,669 @@
...
+  * [見出しとラベル 2.4.6](/WAI/WCAG21/quickref/#headings-and-labels) ([Understanding 2.4.6](/WAI/WCAG21/Understanding/headings-and-labels))
```
Comment (remibetin) [inline _tips/designing.ja.md:359]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 28]
```diff
@@ -0,0 +1,669 @@
...
+          <div class="row error">
+            <label class="required" for="email5">{% include_cached icon.html name="warning" label="Error" %} メールアドレス *</label>
+            <input type="text" id="email5" name="email" value="superbear@@hq.example.com" aria-invalid="true" aria-describedby="error_email">
+          </div>
+          <div class="row">
+            <label for="website3">Webサイト名</label>
+            <input type="text" id="website3" name="website">
+          </div>
+          <div class="row error">
+            <label class="required" for="comment3">{% include_cached icon.html name="warning" label="Error" %} コメント *</label>
```
Comment (remibetin) [inline _tips/designing.ja.md:404]: @jaykishigami Please translate the `label` value

File: _tips/designing.ja.md [hunk 29]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [エラー識別 3.3.1](/WAI/WCAG21/quickref/#error-identification) ([Understanding 3.3.1](/WAI/WCAG21/Understanding/error-identification))
```
Comment (remibetin) [inline _tips/designing.ja.md:422]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 30]
```diff
@@ -0,0 +1,669 @@
...
+  * [ラベルまたは説明書 3.3.2](/WAI/WCAG21/quickref/#labels-or-instructions) ([Understanding 3.3.2](/WAI/WCAG21/Understanding/labels-or-instructions))
```
Comment (remibetin) [inline _tips/designing.ja.md:423]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 31]
```diff
@@ -0,0 +1,669 @@
...
+
+コンテンツ間の関係をより明確にするために､空白と近接を利用する｡見出しをつけることで､コンテンツをグループ化し､乱雑さをなくし､さっと見やすくし理解しやすくする｡
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="例: コンテンツ間の関係を強調するスペーシング" class="example" %}
+{:/}
+
+<div class="headings two-column">
+  <figure class="fail">
+    <figcaption id="inaccessible-headings-title">{% include_cached icon.html name="ex-circle" label="Wrong" %} 間隔が狭く､関係が不明確な場合</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:444]: @jaykishigami Please translate the `label` value

File: _tips/designing.ja.md [hunk 32]
```diff
@@ -0,0 +1,669 @@
...
+          <rect x="0" y="30" width="249" height="3"/>
+          <rect class="image" x="180" y="36" width="70" height="55"/>
+          <line x1="181" y1="37" x2="249" y2="90" stroke-width="1"/>
+          <line x1="181" y1="90" x2="249" y2="37" stroke-width="1"/>
+          <rect x="0" y="36" width="168" height="3"/>
+          <rect x="0" y="42" width="171" height="3"/>
+          <rect x="0" y="48" width="166" height="3"/>
+          <rect x="0" y="54" width="172" height="3"/>
+          <rect x="0" y="60" width="120" height="3"/>
+          <text class="sub-heading" x="0" y="74">Sub heading</text>
```
Comment (remibetin) [inline _tips/designing.ja.md:461]: @jaykishigami "Sub heading" needs translation

File: _tips/designing.ja.md [hunk 33]
```diff
@@ -0,0 +1,669 @@
...
+          <rect x="0" y="76" width="144" height="3"/>
+          <rect x="0" y="82" width="136" height="3"/>
+          <rect x="0" y="88" width="142" height="3"/>
+          <rect x="0" y="94" width="236" height="3"/>
+          <rect x="0" y="100" width="249" height="3"/>
+          <rect x="0" y="106" width="239" height="3"/>
+          <rect x="0" y="112" width="205" height="3"/>
+          <text class="sub-heading" x="0" y="126">Sub heading</text>
```
Comment (remibetin) [inline _tips/designing.ja.md:469]: @jaykishigami "Sub heading" needs translation

File: _tips/designing.ja.md [hunk 34]
```diff
@@ -0,0 +1,669 @@
...
+      </svg>
+    </div>
+  </figure>
+  <figure class="pass">
+    <figcaption id="accessible-headings-title">{% include_cached icon.html name="check-circle" label="OK" %}  間隔を広げ､関係を明確な場合</figcaption>
+    <div>
+      <svg version="1.1" height="173" aria-labelledby="accessible-headings-title" aria-describedby="accessible-headings-desc" role="img">
+        <g>
+          <desc id="accessible-headings-desc">主題, 複数の見出し, リスト, 大きくない分のブロック</desc>
+          <text class="heading" x="0" y="14">Main heading</text>
```
Comment (remibetin) [inline _tips/designing.ja.md:486]: @jaykishigami "Main heading" needs translation

File: _tips/designing.ja.md [hunk 35]
```diff
@@ -0,0 +1,669 @@
...
+          <rect x="0" y="39" width="95" height="3"/>
+          <rect x="0" y="51" width="175" height="3"/>
+          <rect x="0" y="57" width="170" height="3"/>
+          <rect x="0" y="63" width="159" height="3"/>
+          <rect x="0" y="69" width="172" height="3"/>
+          <rect x="0" y="75" width="143" height="3"/>
+          <rect class="image" x="180" y="21" width="70" height="55"/>
+          <line x1="181" y1="22" x2="249" y2="75"/>
+          <line x1="181" y1="75" x2="249" y2="22"/>
+          <text class="sub-heading" x="0" y="98">Sub heading</text>
```
Comment (remibetin) [inline _tips/designing.ja.md:499]: @jaykishigami "Sub heading" needs translation

File: _tips/designing.ja.md [hunk 36]
```diff
@@ -0,0 +1,669 @@
...
+          <rect x="0" y="103" width="106" height="3"/>
+          <rect x="0" y="109" width="99" height="3"/>
+          <rect x="0" y="115" width="114" height="3"/>
+          <rect x="0" y="121" width="110" height="3"/>
+          <rect x="0" y="127" width="110" height="3"/>
+          <rect x="0" y="139" width="99" height="3"/>
+          <rect x="0" y="145" width="114" height="3"/>
+          <rect x="0" y="151" width="110" height="3"/>
+          <rect x="0" y="157" width="84" height="3"/>
+          <text class="sub-heading" x="135" y="98">Sub heading</text>
```
Comment (remibetin) [inline _tips/designing.ja.md:509]: @jaykishigami "Sub heading" needs translation

File: _tips/designing.ja.md [hunk 37]
```diff
@@ -0,0 +1,669 @@
...
+          <rect x="0" y="140" width="243" height="3"/>
+          <rect x="0" y="146" width="250" height="3"/>
+          <rect x="0" y="152" width="245" height="3"/>
+          <rect x="0" y="158" width="187" height="3"/>
+        </g>
+      </svg>
+    </div>
+  </figure>
+  <figure class="pass">
+    <figcaption id="accessible-headings-title">{% include_cached icon.html name="check-circle" label="OK" %}  間隔を広げ､関係を明確な場合</figcaption>
```
Comment (remibetin) [inline _tips/designing.ja.md:481]: @jaykishigami Please translate the `label` value

File: _tips/designing.ja.md [hunk 38]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [見出しとラベル 2.4.6](/WAI/WCAG21/quickref/#headings-and-labels) ([Understanding 2.4.6](/WAI/WCAG21/Understanding/headings-and-labels))
```
Comment (remibetin) [inline _tips/designing.ja.md:534]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 39]
```diff
@@ -0,0 +1,669 @@
...
+  * [章の見出し 2.4.10](/WAI/WCAG21/quickref/#section-headings) ([Understanding 2.4.10](/WAI/WCAG21/Understanding/section-headings))
```
Comment (remibetin) [inline _tips/designing.ja.md:535]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 40]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [文字でないコンテンツ 1.1.1](/WAI/WCAG21/quickref/#non-text-content) ([Understanding 1.1.1](/WAI/WCAG21/Understanding/non-text-content))
```
Comment (remibetin) [inline _tips/designing.ja.md:609]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 41]
```diff
@@ -0,0 +1,669 @@
...
+{::nomarkdown}
+{% include_cached box.html type="end" %}
+{:/}
+
+{::nomarkdown}
+{% include_cached box.html type="start" title="詳細情報" class="simple" %}
+{:/}
+
+* **WCAG**
+  * [オーディオコントロール 1.4.2](/WAI/WCAG21/quickref/#audio-control) ([Understanding 1.4.2](/WAI/WCAG21/Understanding/audio-control))
```
Comment (remibetin) [inline _tips/designing.ja.md:644]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 42]
```diff
@@ -0,0 +1,669 @@
...
+  * [一時停止､停止､離れる 2.2.2](/WAI/WCAG21/quickref/#pause-stop-hide) ([Understanding 2.2.2](/WAI/WCAG21/Understanding/pause-stop-hide))
```
Comment (remibetin) [inline _tips/designing.ja.md:645]: @jaykishigami Please translate "Understanding"

File: _tips/designing.ja.md [hunk 43]
```diff
@@ -0,0 +1,669 @@
...
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple translators
+# contributors:
+# - name: "Jan Doe"   # Replace Jan Doe with contributor name, or delete this line if none
+# - name: "Jan Doe"   # Replace Jan Doe with name, or delete this line if not multiple contributors
+
+github:
+  repository: w3c/wai-quick-start
+  path: '_tips/designing.md'   # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+
+permalink: /tips/designing/   # Add the language shortcode to the end, with no slash at the end. For example /path/to/file/fr
```
Comment (remibetin) [inline _tips/designing.ja.md:29]: @jaykishigami Please add the language shortcode to the end of the permalink, to make the translation appear in the header.

See https://www.w3.org/WAI/about/translating/guides/new-translation/ for more guidance.

**Review by remibetin — APPROVED** [review]

**Comment by shawna-slh** [comment]
Thank you @jaykishigami and @remibetin !

**Comment by remibetin** [comment]
QA done: OK ✅

**Pull request merged**